app/src/main/java/edu/uc/hickmadc/customnotifications/MainActivity.kt

```diff
@@ -9,13 +9,12 @@ import androidx.navigation.ui.NavigationUI
 import com.google.android.material.bottomnavigation.BottomNavigationView
 import edu.uc.hickmadc.customnotifications.ui.main.MyListAdapter
 
-//import edu.uc.hickmadc.customnotifications.ui.main.MainFragment
```

Comment:
I removed most of the necessarily comments so that the code doesn't get too confusing and easier to read.

Reply:
Source: https://rules.sonarsource.com/kotlin/RSPEC-125